website/blog/2021-04-20-2.3.0.md

```diff
@@ -0,0 +1,1728 @@
+<!-- prettier-ignore -->
+```hbs
+{{!-- Input --}}
+<div class="
+foo"></div>
+<div bar='"{{expr}}"'></div>
+<div baz="\{{ non-expression }}"></div>
+
+{{!-- Prettier 2.2 --}}
+<div class="[object Object],foo"></div>
+<div bar=""{{expr}}""></div>
+<div baz="{{ non-expression }}"></div>
+
+{{!-- Prettier 2.3 --}}
+<div class="foo"></div>
+<div bar='"{{expr}}"'></div>
+<div baz="\{{ non-expression }}"></div>
+```
+
+#### Split text content on multiple lines ([#10179](https://github.com/prettier/prettier/pull/10179) by [@dcyriller](https://github.com/dcyriller))
+
+<!-- prettier-ignore -->
+```handlebars
+{{!-- Input --}}
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more and more.
+</div>
+
+{{!-- Prettier 2.2 --}}
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more and more.
+</div>
+
+{{!-- Prettier 2.3 --}}
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more
+  and more.
+</div>
+```
+
+#### Preserve numeric character references ([#10550](https://github.com/prettier/prettier/pull/10550) by [@rwjblue](https://github.com/rwjblue) and [@thorn0](https://github.com/thorn0))
+
+<!-- prettier-ignore -->
+```hbs
+{{! Input }}
+<span class="stampFont" style="font-family: 'stampfont'">&#xf000;</span>
+
+{{! Prettier 2.2 }}
+<span class="stampFont" style="font-family: 'stampfont'"></span>
+
+{{! Prettier 2.3 }}
+<span class="stampFont" style="font-family: 'stampfont'">&#xf000;</span>
+```
+
+#### Do not break opening mustache and path ([#10586](https://github.com/prettier/prettier/pull/10586) by [@dcyriller](https://github.com/dcyriller))
+
+<!-- prettier-ignore -->
+```handlebars
+{{!-- Input --}}
+<GlimmerComponent
+  @errors={{or this.aVeryLongProperty (and this.aProperty (v-get bike "number" "message"))}}
+  data-test-beneficiary-account-number
+/>
+<GlimmerComponent
+  @progress={{aPropertyEngdingAfterEightiethColumnToHighlightAWeirdClosingParenIssue}}
+/>
+
+{{!-- Prettier 2.2 --}}
+<GlimmerComponent
+  @errors={{
+    or
+    this.aVeryLongProperty
+    (and this.aProperty (v-get bike "number" "message"))
+  }}
+  data-test-beneficiary-account-number
+/>
+<GlimmerComponent
+  @progress={{
+    aPropertyEngdingAfterEightiethColumnToHighlightAWeirdClosingParenIssue
+  }}
+/>
+
+{{!-- Prettier 2.3 --}}
+<GlimmerComponent
+  @errors={{or
+    this.aLongProperty
+    (and this.aProperty (v-get bike "number" "message"))
+  }}
+  data-test-beneficiary-account-number
+/>
+<GlimmerComponent
+  @progress={{aPropertyEngdingAfterEightiethColumnToHighlightAWeirdClosingParenIssue}}
+/>
+```
+
+### GraphQL
+
+#### Fix missing space after keyword in anonymous operations ([#10689](https://github.com/prettier/prettier/pull/10689) by [@patriscus](https://github.com/patriscus))
+
+<!-- prettier-ignore -->
+```graphql
+# Input
+query ($unnamed: String) {
+  id
+}
+
+# Prettier 2.2
+query ($unnamed: String) {
+  id
+}
+
+# Prettier 2.3
+query($unnamed: String) {
```

Comment:
This looks like it removes the space — is the example code correct or should the title be changed?